install/martypc.toml

```diff
@@ -21,7 +21,7 @@ fuzzer = false
 # Run the emulated machine as fast as possible. Note this isn't just a CPU 
 # boost, the entire system including timer will accellerate. 
 # NOTE: This will cause unstable FPS. Mostly used for debugging.
-warpspeed = false 
+warpspeed = true
```

Comment:
this runs the entire machine as fast as the host can possibly run it; this includes the timer and sound and video and everything, it really isn't an appropriate default